apertium-tur.tur.lexc

```diff
@@ -4071,6 +4071,7 @@ kıssa:kıssa N1 ; !
 seç:seç N1 ; !
```

Comment:
seç is not "noun", it is verb v tv, but "saç"=hair is a "noun"

Reply:
Just deleted that line, because both below exists already:
seç:seç V-AR-TV ; ! "choose" (DIR)
saç:saç N1 ; ! "hair"